ops/platform/000-bootstrap/codebuild.tf

```diff
@@ -26,8 +42,36 @@ locals {
       image        = "aws/codebuild/amazonlinux-aarch64-standard:3.0"
       privileged   = true # Enables this Runner to build Docker images
       timeout      = 60
+      vpc_config   = []
     }
   }
+  env_vpc = local.account_type == "non-prod" ? "bfd-east-test" : "bfd-prod-test"
+}
+
+data "aws_vpc" "this" {
+  filter {
+    name   = "tag:Name"
+    values = [local.env_vpc]
+  }
+}
+
+data "aws_subnets" "private" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.this.id]
+  }
+  filter {
+    name   = "tag:use"
+    values = ["private"]
+  }
+}
+
+data "aws_security_group" "this" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.this.id]
+  }
+  name = "cms-cloud-security-validation-egress"
```

Comment:
I wish we could rely on the terraservice module for these, but I recognize some of the challenges here and haven't given it enough thought–the `env` here is `platform` which won't have an obvious way to lookup the _real_ path-to-production, env-specific resources like the vpc. 🤔

Reply:
@malessi Going ahead with a separate one... Thanks!